test/testdata/schema/ghc92/vscode-extension-schema.golden.json

```diff
@@ -249,6 +249,622 @@
         "scope": "resource",
         "type": "boolean"
     },
+    "haskell.plugin.semanticTokens.config.class": {
+        "default": "class",
+        "description": "LSP semantic token type to use for Class",
+        "enum": [
+            "namespace",
+            "type",
+            "class",
+            "enum",
+            "interface",
+            "struct",
+            "typeParameter",
+            "parameter",
+            "variable",
+            "property",
+            "enumMember",
+            "event",
+            "function",
+            "method",
+            "macro",
+            "keyword",
+            "modifier",
+            "comment",
+            "string",
+            "number",
+            "regexp",
+            "operator",
+            "decorator"
+        ],
+        "enumDescriptions": [
+            "LSP Semantic Token Type:namespace",
```

Comment:
these are kind of pointless actually, aren't they? maybe just skip them since they don't actually say anything new

Reply:
Yes, but we should leave it here better than the uggly list of empty string.

Reply:
Oh, I didn't realise they were mandatory. Then I guess its fine